sinks/splunk/splunk.go

```diff
@@ -464,6 +469,13 @@ func (sss *splunkSpanSink) Ingest(ssfSpan *ssf.SSFSpan) error {
 		Name:           ssfSpan.Name,
 	}
 
+	if ssfSpan.Indicator {
```

Comment:
should we just set this on all spans? Worried that the two conditionals (above and here) will diverge via bitrot. It's more bytes, but would help discoverability / reduce head-scratching when you find spans without the `partial` field... and it's more symmetric to the `indicator` field (which we could also drop from non-indicators).

Reply:
It will only ever be true on non-indicators, so it's not useful as a filter. Every byte reduces our retention window, so I think it's worth the savings both for logical and practical reasons.

Reply:
What about just setting variables for both, e.g. `wouldDrop` and `doDrop`, to keep the logic in one place and the outcome in the other?

Reply:
yep, I'd like that! Let's do that!

Reply:
Updated

Reply:
I think this should be 

```suggestion
	if wouldDrop && !actuallyDrop {
```

now?

Reply:
I don't think so? "If we decided to drop it, drop it" -- not "If we would have sample-dropped it and didn't decide to drop it, drop it"

Reply:
I think `ssfSpan.Indicator` more accurately represents the intent here: We only add this field to indicator spans?

Reply:
From above, I feel these two conditionals will diverge via bitrot. What I think this should do is to add the `partial` field to any span that made it in despite the trace not having made it... no matter if it's an indicator or not (there might be other things in the future, who knows?)

Reply:
(This would also only put the value on indicator spans when it's true, as well as putting it on all non-indicator spans)

Reply:
`!actuallyDrop` is redundant here actually, so `if wouldDrop {` works as intended I think. It's unfortunate we can't encode the false case, but I'll take the byte savings and can't think of any reason to need to easily search when it's false